.github/workflows/ci.yml

```
@@ -29,8 +33,9 @@ jobs:
         NXF_VER:
           - "24.04.2"
           - "latest-everything"
+        tags:
+          - "tomte"
```

Add one tag for the skip_drop test, and another tag for the "skip many things"-test :) 

```suggestion
          - "tomte"
```

---

Done